src/w3d/renderer/mesh.cpp

```diff
@@ -343,7 +343,10 @@ void MeshClass::Render_Material_Pass(MaterialPassClass *pass, IndexBufferClass *
 
         MultiListIterator<DX8PolygonRendererClass> it(&(m_model->m_polygonRendererList));
```

Comment:
is something missing in this loop? there are Draw_Strip Draw_Triangles calls in wb